pkg/scheduler/framework/session_plugins.go

```diff
@@ -154,6 +155,9 @@ func (ssn *Session) Preemptable(preemptor *api.TaskInfo, preemptees []*api.TaskI
 				continue
 			}
 			candidates := pf(preemptor, preemptees)
+			if candidates == nil {
+				break
```

Comment:
continue?

Reply:
Within one Tier, each plugin preemptFn get victims and finally get intersections.  Here if one plugin get no victim, the intersection must be nil, no need to continue.

Reply:
ok, ic.  I didn't understand the tiers concept.

Reply:
I think we define return values as follow:

1. `nil` means this plugin can not make decission
1. `empty list` means this plugin rejects all victims
1. `non-empty list` means this plugin accept victims in the list

So we can not break if it's `nil` :)

Reply:
@k82cn 

The solution has been updated. Please see the "Solution Update" description and help to review.